src/mods.cpp

```diff
@@ -61,28 +60,65 @@ void parseModContents(ModSpec &spec)
 
 	// Handle modpacks (defined by containing modpack.txt)
 	std::ifstream modpack_is((spec.path+DIR_DELIM+"modpack.txt").c_str());
-	if(modpack_is.good()){ //a modpack, recursively get the mods in it
+	if (modpack_is.good()) { //a modpack, recursively get the mods in it
```

Comment:
This confused me, is "is" for "input stream" a common thing? I'd prefer modpack_file.